examples/shared_samples/fcoe_network.rb

```diff
@@ -36,7 +36,7 @@
 fcoe_network_class = OneviewSDK.resource_named('FCoENetwork', @client.api_version)
 
 # Scope class used in this sample
-scope_class = OneviewSDK.resource_named('Scope', @client.api_version) unless @client.api_version.to_i <= 200
+scope_class = OneviewSDK.resource_named('Scope', 1200) unless @client.api_version.to_i <= 200
```

Comment:
Please remove this hardcoded apiversion.

Reply:
Done